packages/applications-service-api/src/services/application.service.js

```diff
@@ -1,21 +1,45 @@
 const {
-	getByCaseReference: getBackOfficeApplication
-} = require('../repositories/project.repository');
+	getByCaseReference: getBackOfficeApplication,
+	getAllApplications: getAllBOApplicationsRepository
+} = require('../repositories/project.backoffice.repository');
+const { getAllNIApplications } = require('./application.ni.service');
 const { getNIApplication } = require('./application.ni.service');
 const config = require('../lib/config');
 const {
 	mapNIApplicationToApi,
-	mapBackOfficeApplicationToApi
+	mapBackOfficeApplicationToApi,
+	mapBackOfficeApplicationsToApi
 } = require('../utils/application.mapper');
 
 const getApplication = async (caseReference) =>
 	isBackOfficeApplication(caseReference)
 		? mapBackOfficeApplicationToApi(await getBackOfficeApplication(caseReference))
 		: mapNIApplicationToApi(await getNIApplication(caseReference));
 
+const getAllApplications = async (query) => {
```

Comment:
This is great - deciding in the service is going to be really useful as we may need to extend this to read from both NI and BO and merge results.